TwitchLib.Client.Models/TwitchLibMessage.cs

```diff
@@ -0,0 +1,41 @@
+﻿#if NETSTANDARD
+    using TwitchLib.Client.Models.Extensions.NetCore;
+#endif
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using TwitchLib.Client.Enums;
+    using TwitchLib.Client.Models.Internal;
+
+#if NET452
+
+#endif
+
+namespace TwitchLib.Client.Models
+{
+    /// <summary>Class represents Message.</summary>
+    public abstract class TwitchLibMessage
```

Comment:
Is `TwitchLibMessage` a good name? Can we get away with something like `Message`?